flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/inference/strategies/JsonQueryOnErrorEmptyArgumentTypeStrategy.java

```diff
@@ -0,0 +1,57 @@
+package org.apache.flink.table.types.inference.strategies;
```

Comment:
It seems this class is without license and is a result of rat-check failure

Reply:
I don’t have my template for new files properly set. I’ll fix that. Any other comments?